include/rx/ranges.hpp

```diff
@@ -763,43 +764,6 @@ template <class T>
     return repeat(std::forward<T>(v));
 }
 
-/*!
-    @brief Generate N copies of type T.
-
-    This is equivalent to `fill(value) | take(n)`.
-*/
-template <class T>
-struct repeat_n {
-    using output_type = const T&;
-    static constexpr bool is_finite = true;
-    static constexpr bool is_idempotent = true;
-
-    T value;
-    size_t n;
-    size_t i = 0;
-    constexpr explicit repeat_n(size_t n, T value) : value(std::move(value)), n(n) {}
-
-    constexpr void next() {
-        ++i;
-    }
-    [[nodiscard]] constexpr output_type get() const noexcept {
-        return value;
-    }
-    [[nodiscard]] constexpr bool at_end() const noexcept {
-        return i == n;
-    }
-    constexpr size_t size_hint() const noexcept {
-        return n;
-    }
-};
-template <class T>
-repeat_n(size_t, T &&)->repeat_n<T>;
-
-template <class T>
-[[nodiscard]] constexpr auto fill_n(size_t n, T&& v) noexcept {
-    return repeat_n(n, std::forward<T>(v));
-}
-
```

Comment:
Why was this removed?

Reply:
It was moved further to the bottom, because it can only be defined after struct take: https://github.com/simonask/rx-ranges/pull/17/files#diff-d9c605d1767259c15ff78a379ec1212bR955

Reply:
Oh sorry, missed that.